datahub-web-react/src/app/search/EditTextModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Input, Modal } from 'antd';
+import React, { useState } from 'react';
+
+type Props = {
+    onCloseModal: () => void;
+    onOkOverride?: (result: string) => void;
+    title?: string;
+    defaultValue?: string;
+};
+
+export const EditTextModal = ({ defaultValue, onCloseModal, onOkOverride, title }: Props) => {
+    const [stagedValue, setStagedValue] = useState(defaultValue || '');
+    return (
+        <Modal
+            title={title}
+            visible
+            onCancel={onCloseModal}
+            keyboard
+            footer={
+                <>
+                    <Button onClick={onCloseModal} type="text">
+                        Cancel
+                    </Button>
+                    <Button disabled={stagedValue.length === 0} onClick={() => onOkOverride?.(stagedValue)}>
```

Comment:
this wouldn't be disabled if there is whitespace then right? might want to check if `stadedValue.trim().length === 0`